experimenter/experimenter/experiments/api/v6/urls.py

```diff
@@ -18,4 +20,12 @@
     NimbusExperimentDraftViewSet,
     "nimbus-experiment-draft-rest",
 )
+
 urlpatterns = router.urls
+urlpatterns += [
+    re_path(
+        r"nimbus-fml-diagnostics",
+        NimbusFmlDiagnosticsApi.as_view(),
+        name="nimbus-fml-diagnostics-api",
+    ),
+]
```

Comment:
Should this use the `router.register` more directly?

```suggestion
router.register(
    r"nimbus-fml-diagnostics",
    NimbusFmlDiagnosticsApi.as_view(),
    name="nimbus-fml-diagnostics-api",
)
```

Reply:
`ApiView`s cannot be registered with a router in the same way that other views can -- it needs to be added as a path without the router